app/src/main/java/com/droidcba/kedditbysteps/features/news/adapter/NewsDelegateAdapter.kt

```diff
@@ -22,16 +27,17 @@ class NewsDelegateAdapter : ViewTypeDelegateAdapter {
         holder.bind(item as RedditNewsItem)
     }
 
-    class NewsViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
+    inner class NewsViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
             parent.inflate(R.layout.news_item)) {
 
         fun bind(item: RedditNewsItem) = with(itemView) {
-            //Picasso.with(itemView.context).load(item.thumbnail).into(img_thumbnail)
             img_thumbnail.loadImg(item.thumbnail)
             description.text = item.title
             author.text = item.author
             comments.text = "${item.numComments} comments"
             time.text = item.created.getFriendlyTime()
+
+            super.itemView.setOnClickListener( {mCommit.onItemSelected(item.url)} )
```

Comment:
As the last parameter is a function, you can remove the brackets in this way:
`super.itemView.setOnClickListener { mCommit.onItemSelected(item.url) }`